### PR TITLE
ClickHouse and Metabase tutorial

### PR DESCRIPTION
**Pull request by anelook**
This is a tutorial on integrating Metabase with ClickHouse

Here is the direct link to rendered tutorial -  https://ch-tutorial.devportal.pages.dev/docs/tutorials/clickhouse-metabase

### CODE REVIEW

File: docs/tutorials/clickhouse-metabase.rst [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+.. code:: bash
+
+    docker run -d -p 3000:3000 --name metabase metabase/metabase
+
+.. code:: bash
+
+    docker run -d -p 3000:3000 \
+    --mount type=bind,source=/path-to-plugins-folder,destination=/plugins \
+    --name metabase metabase/metabase
```
Comment (TibsAtWork) [inline docs/tutorials/clickhouse-metabase.rst:55]: Do we only need the second of these examples?
And maybe explain briefly that `/path-to-plugins-folder` is the path you want *outside* docker?

File: docs/tutorials/clickhouse-metabase.rst [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+.. image:: /images/tutorials/clickhouse-metabase/docker.gif
+   :alt: Animated GIF showing pulling latest docker image and running it
```
Comment (TibsAtWork) [inline docs/tutorials/clickhouse-metabase.rst:58]: This doesn't seem to play in the preview.
Maybe put a line in front of it saying "Here is a preview of how that should work" ?

File: docs/tutorials/clickhouse-metabase.rst [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+Adding drivers
+++++++++++++++
+Metabase comes with a predefined set of supported database. ClickHouse is not in `the list of officially supported drivers <https://www.metabase.com/docs/latest/databases/connecting#connecting-to-supported-databases>`_, but we can include it `as a third-party community driver <https://www.metabase.com/docs/latest/developers-guide/partner-and-community-drivers#community-drivers>`_.
+
+You can find ClickHouse driver (and extra documentation, in case you need it) in ClickHouse driver for Metabase `GitHub repository <https://github.com/ClickHouse/metabase-clickhouse-driver>`_.
+
+Load the driver by going `to the latest release <https://github.com/ClickHouse/metabase-clickhouse-driver/releases>`_ and selecting ``clickhouse.metabase-driver.jar`` from the list of assets.
+
+.. image:: /images/tutorials/clickhouse-metabase/driver.gif
+   :alt: Animated GIF showing loading the driver for ClickHouse Metabase from the GitHub page
```
Comment (TibsAtWork) [inline docs/tutorials/clickhouse-metabase.rst:69]: Whereas this GIF does play in the preview

File: docs/tutorials/clickhouse-metabase.rst [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+
+Using the visual editor
++++++++++++++++++++++++
+
+Alternatively, you can create a query using Metabase visual editor. For instance, out of curiosity we'll look at dishes that were popular before 1920, but then disappeared from the menu.
+
+Press on the **New** button and select **Question** from the available options.
+
+Metabase will ask you to pick the database and the table to run requests. Select **AivenForClickHouse** (or the name you gave to your database) and ``Menu Item Denorm``. Once data is selected you'll lend on the wizard with multiple options to shape the request.
```
Comment (TibsAtWork) [inline docs/tutorials/clickhouse-metabase.rst:129]:
```suggestion
Metabase will ask you to pick the database and the table to run requests. Select **AivenForClickHouse** (or the name you gave to your database) and ``Menu Item Denorm``. Once data is selected you'll use the wizard with multiple options to shape the request.
```
`lend on` doesn't work for me? Did you mean `lean on`? Or perhaps just `use`.

**Review by TibsAtWork — COMMENTED** [review]
I really like this.
I've made some suggestions (and will fix a few small things in a commit)

File: docs/tutorials/clickhouse-metabase.rst (hunk 2 above)
Comment (anelook) [reply docs/tutorials/clickhouse-metabase.rst:56]: Interesting, I see them all playing in the preview. Can it be that the image was still loading?

**Comment by anelook** [comment]
Added the minor fixes! All gifs load for me and work well in preview.

**Comment by TibsAtWork** [comment]
This is now at https://aiven.io/developer/clickhouse-metabase and so this PR can be closed

**Pull request closed without merging**